app/services/reset_user_password.rb

```diff
@@ -1,20 +1,30 @@
 class ResetUserPassword
-  def initialize(user:, log_stdout: false)
+  def initialize(user:)
     @user = user
-    @log_stdout = log_stdout
   end
 
   def call
-    reset_user_password_and_log_event
+    reset_user_password
+    log_event
+    notify_user
   end
 
   private
 
   attr_reader :user
 
-  def reset_user_password_and_log_event
+  def reset_user_password
     user.update!(password: SecureRandom.hex(8))
-    return unless @log_stdout
```

Comment:
the STDOUT log was part of the old `bin/` script and `rake` tasks. If anybody feels strongly, I can bring them back. But they haven't been used in a while so I figured we might as well remove them